Digipost/build.gradle

```diff
@@ -96,7 +96,7 @@ dependencies {
     implementation('com.sun.jersey:jersey-json:1.19.3') {
         exclude module: 'jaxb-impl'
     }
-    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
compile is deprecated, use implementation
See; https://developer.android.com/studio/build/dependencies

Reply:
👍 :)